pkg/utils/grpcutil/grpcutil_test.go

```diff
@@ -41,9 +41,9 @@ func TestToTLSConfig(t *testing.T) {
 
 	re := require.New(t)
 	tlsConfig := TLSConfig{
-		KeyPath:  path.Join(certPath, "pd-server-key.pem"),
-		CertPath: path.Join(certPath, "pd-server.pem"),
-		CAPath:   path.Join(certPath, "ca.pem"),
+		KeyPath:  filepath.Join(certPath, "pd-server-key.pem"),
```

Comment:
Why change only `a part of` it? It looks like there's still quite a bit of `path` used

Reply:
some of them are etcd keys which doesn't rely on os

Reply:
got it.